packages/data-context/src/actions/BrowserActions.ts

```diff
@@ -44,6 +44,14 @@ export class BrowserActions {
     await this.browserApi.focusActiveBrowserWindow()
   }
 
+  resetFocusIfMacOS () {
```

Comment:
/nit: Personal preference, I would put the "this is a workaround" and GH issue link on the function as JSDoc